.env

```diff
@@ -62,7 +63,7 @@ _APP_MAINTENANCE_RETENTION_EXECUTION=1209600
 _APP_MAINTENANCE_RETENTION_ABUSE=86400
 _APP_MAINTENANCE_RETENTION_AUDIT=1209600
 _APP_MAINTENANCE_RETENTION_SCHEDULES=86400
-_APP_USAGE_TIMESERIES_INTERVAL=60
+_APP_USAGE_TIMESERIES_INTERVAL=20
```

Comment:
why did we reduce this ?

Reply:
There is sleep 20 in test and only 1 retry. That would create 40 seconds time for test to pass, leading remaining 20 secs possible to fail test for no reason.
Decreasing this should resolve it.

Also IIRC, this was something like 5 seconds at some point. We can sync with Damodar to see if 20 is fine for development.

Reply:
Okay 👍